src/lib/token.h

```diff
@@ -54,8 +54,6 @@ struct sealobject {
     twist sopub;
     twist sopriv;
     twist soauthsalt;
-
-    uint32_t handle;
```

Comment:
Can you also (not in this series per-say) deal with the TODO on l75:
https://github.com/tpm2-software/tpm2-pkcs11/pull/493/files#diff-e81270e2eb0c861a9aad8a2ded26dad7R75

Reply:
That's what I'm working towards with this series... :-)